### ISSUE TITLE
Touch Right click Crash

### ISSUE THREAD

**Issue by arcandio** [typora/typora-issues#590]
> On Windows 10
Microsoft Surface Pro 4
Typora version 0.9.25(beta)

Typora crashes reliably when touch-holding on ~~links~~ anything. Using a mouse to right-click works fine.

**Comment by abnerlee** [comment]
I tried "Duet" to make my iPad a touchscreen, but right click by two finger click works fine.

So, how you do right click on Surface Pro

And, could you send the `typora.log` under `C:\Users\{your-user-name}\AppData\Roaming\Typora` to <hi@typora.io>?

Thanks for the report @arcandio

**Comment by arcandio** [comment]
No prob, the log's sent.

On a Surface, you touch-and-hold to do a right click, and it seems to bring up a slightly different UI from the usual windows right-click UI, specifically designed for touch, I would think.

I did manage to get a right-click off on an empty typora screen, but when I typed into the document and selected text, then touched-and-held for a right-click, it crashed as usual.

Good luck on this one, it seems like something that might be a problem with the Surface Windows ui system or something.
